pkg/envoy/cds/cluster.go

```diff
@@ -34,18 +33,12 @@ var replacer = strings.NewReplacer(".", "_", ":", "_")
 
 type clusterOptions struct {
 	permissive             bool
-	withTLS                bool
 	withActiveHealthChecks bool
 }
 
 // clusterOption is type of function that edits the defaults of the options struct.
 type clusterOption func(o *clusterOptions)
 
-// withTLS is an option to disable TLS.
-func withTLS(o *clusterOptions) {
-	o.withTLS = true
-}
-
```

Comment:
I wonder what the origin of this function is and what we intended with it?
Glad we are removing it - we are always-mTLS mesh.

Reply:
I think it was added by Sean, when we was trying to build the upstream clusters for the gateway. I too wasn't sure what the intention was, since its not needed I removed it